google/cloud/bigtable/table_admin.h

```diff
@@ -1389,7 +1389,29 @@ class TableAdmin {
    * @par Example
    * @snippet table_admin_iam_policy_snippets.cc get iam policy
    */
+
   StatusOr<google::iam::v1::Policy> GetIamPolicy(std::string const& table_id);
+  /**
+   * Gets the policy for @p backup_id.
+   *
+   * @param clusterid_id the associated cluster that contains backup.
+   *
+   * @param backup_id the backup to query.
+   * @return google::iam::v1::Policy the full IAM policy for the backup.
+   *
+   * @par Idempotency
+   * This operation is read-only and therefore it is always idempotent.
+   *
+   * @par Thread-safety
+   * Two threads concurrently calling this member function on the same instance
+   * of this class are **not** guaranteed to work. Consider copying the object
+   * and using different copies in each thread.
+   *
+   * @par Example
+   * @snippet table_admin_iam_policy_snippets.cc get iam policy
```

Comment:
this `get iam policy` snippet is not for backup, so either we write a snippet for `get iam policy bckup` or at least create a todo.